src/webapp/components/main-layout/MainLayout.tsx

```diff
@@ -6,8 +6,16 @@ import { SideBar } from "../sidebar/SideBar";
 import ExitToAppIcon from "@material-ui/icons/ExitToApp";
 import i18n from "@eyeseetea/d2-ui-components/locales";
 import { AppFooter } from "../app-footer/AppFooter";
+import { goToDhis2Url } from "../../utils/helpers";
+import { useConfig } from "@dhis2/app-runtime";
 
 export const MainLayout: React.FC = React.memo(({ children }) => {
+    const { baseUrl } = useConfig();
+
+    const handleLogout = () => {
+        goToDhis2Url(baseUrl, "/dhis-web-commons-security/logout.action");
```

Comment:
There was recommendation on earlier PR, which haven't been merged yet. To just use 'logout' instead of 'handleLogout' for component methods. I'll approve this PR for merging. 
And I will just apply the above recommendation to the commit that tackles the said issue. on later commits. To avoid conflict.